flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/MaterializedTableStatementITCase.java

```diff
@@ -236,6 +241,80 @@ void testCreateMaterializedTableInFullMode() {
                         "Only support create materialized table in continuous refresh mode currently.");
     }
 
+    @Test
+    void testAlterMaterializedTableRefresh() throws Exception {
+        // initialize session handle, create test-filesystem catalog and register it to catalog
+        // store
+        SessionHandle sessionHandle = initializeSession();
+
+        String materializedTableDDL =
+                "CREATE MATERIALIZED TABLE users_shops"
+                        + " PARTITIONED BY (ds)\n"
+                        + " WITH(\n"
+                        + "   'format' = 'debezium-json'\n"
+                        + " )\n"
+                        + " FRESHNESS = INTERVAL '30' SECOND\n"
+                        + " AS SELECT \n"
+                        + "  user_id,\n"
+                        + "  shop_id,\n"
+                        + "  ds,\n"
```

Comment:
I think we should control the ds value generated by datagen, then, so we can the result for the refresh statement.

Reply:
use values source to check data overwrite